docs/body.md

```diff
@@ -252,7 +252,7 @@ Choose that option and you'll be presented with a dialog asking for the
 following inputs.
 
 * Device To Create - non-empty, non-ip, non-dns, unique name to use for this device in Zenoss. ''See note below''.
-* Auth URL - A keystone URL, such as http://\<hostname\>:5000/v2.0/ for Keystone v2, or http://\<hostname\>:5000/v3/ for Keystone v3.
+* Auth URL - A keystone URL, such as http://\<hostname\>:5000/ (latest supported API version will be selected by default, to force an API version, add the path to the end of the URL, like http://\<hostname\>:5000/v3/ or http://\<hostname\>:5000/v2.0/).
```

Comment:
Okay, I can reword it.  The main reason I can see not to autodetect it is the performance impact of doing the detection, but after the first time, we store the URL, so it shouldn't matter.  Is there another reason not to autodetect?